index.html

```diff
@@ -0,0 +1,115 @@
+                            forgot my User ID or Password</a></span>
+                </div>
+            </div>
+
+            <button type="button" class="signUp-button"><i class="material-icons lock">lock</i><span class="SignIn-text">Sign In</span></button>
+
+        </header>
+
+<!-------------------- L O G O  A N D  I C O N S ----------------------->
+
+        <main class="main-cont">
+
+            <section class="logo-icons-cont">
+
+                <div class="logo-container">
+                    <img src="images/intuit-brand1.png" alt="intruit-logo" class="intuit-brand-logo">
+                </div>
+
+                <div class="three-icons-cont">
+                    <div>
+                        <a href="https://www.mint.com/" target="_blank"><img src="images/mint-img.png" alt="mint-icon" class="mint-brand-logo"></a>
+                    </div>
+                    <div>
+                        <a href="https://quickbooks.intuit.com/" target="_blank"><img src="images/quickbooks-img.png" alt="quickbooks-icon" class="quickbooks-brand-logo"></a>
+                    </div>
+                    <div>
+                        <a href="https://turbotax.intuit.com/" target="_blank"><img src="images/turbotax-img.png" alt="turbotax-icon" class="turbotax-brand-logo"></a>
+                    </div>
+                </div>
+
+            </section>
+
+<!------------------------------ F O R M --------------------------------->
+
+            <section class="mainForm-cont">
+
+                <span class="intuitHead">Create an Intuit account</span>
+                <span class="intuitAcount">One account for everything Intuit, including Mint.</span>
+                <span class="learn-more"><a href="https://accounts-help.intuit.com/app/intuit/1995107" target="_blank">Learn more</a></span>
+
+                <form class="form-cont">
+
+                    <span class="email">Email Address</span><br>
+                    <input type="text" class="emailInput"><br><br><br>
+                    <span class="phone">Phone (recommended)</span><br>
+                    <input type="text" class="phoneInput"><br>
+                    <span class="phone-alert">Standard call, messaging or data rates may apply.</span><br><br><br>
+                    <span class="password">Password</span><br>
+                    <input type="text" class="passwordInput"><span class="material-icons unlock-icon">lock_open</span><br><br>
+
+                    <button type="button" class=account-btn><i class="material-icons lock">lock</i>Create Account</button>
+                
+                </form>
```

Comment:
You should use `label` tag in the forms, it is more appropriate, please modify it.